src/psyclone/tests/psyGen_test.py

```diff
@@ -2682,8 +2679,24 @@ def test_codeblock_can_be_printed():
     assert "]" in str(cblock)
 
 
+def test_codeblock_getstatements():
+    '''Test that the get_statements method of a CodeBlock instance returns
+    a copy of the list of nodes from the original AST that are associated with
+    this code block.
+
+    For simplicity we use a list of strings rather than an AST.
+
+    '''
+    original = ["hello", "there"]
+    cblock = CodeBlock(original)
+    result = cblock.get_statements
+    assert result == original
+    # Check that the list is a copy not a reference.
+    assert id(result) != id(original)
```

Comment:
I learned something here :)

Reply:
Andy and Sergi were impressed that you learnt something from me until I told them what it was and they pointed out that I should use "is", which is better, so I've changed it. :-)